### PR TITLE
PXP-8318 Split public and private audit configs

### PR DESCRIPTION
**Pull request by paulineribeyre**
Jira Ticket: [PXP-8318](https://ctds-planx.atlassian.net/browse/PXP-8318)

### New Features
- Split the private fields from the public fields in the audit logging configuration

### CODE REVIEW

File: src/audit/pull_from_queue.py [hunk 1]
```diff
@@ -81,11 +81,27 @@ async def pull_from_queue_loop():
     """
     logger.info("Starting to pull from queue...")
     aws_sqs_config = config["QUEUE_CONFIG"]["aws_sqs_config"]
+    # we know the cred is in AWS_CREDENTIALS (see `AuditServiceConfig.validate`)
```
Comment (themarcelor) [inline src/audit/pull_from_queue.py:84]: Should we consider more logger.info or logger.debug lines here to facilitate some debugging later down the line?

**Review by themarcelor — APPROVED** [review]
+1

File: src/audit/pull_from_queue.py (hunk 1 above)
Comment (paulineribeyre) [reply src/audit/pull_from_queue.py:84]: as mentioned in the comment, it's already checked in the config validation, so the service would not start if it's misconfigured. when we reach this point there shouldn't be anything to log

File: src/audit/pull_from_queue.py [hunk 2]
```diff
@@ -81,11 +81,27 @@ async def pull_from_queue_loop():
...
+    aws_creds = (
+        config["AWS_CREDENTIALS"][aws_sqs_config["aws_cred"]]
+        if "aws_cred" in aws_sqs_config
```
Comment (nss10) [inline src/audit/pull_from_queue.py:87]: shouldn't this be `if "aws_cred" in aws_sqs_config and aws_sqs_config["aws_cred"]`? I mean what if we have an empty value for `aws_sqs_config["aws_cred"]` like in -- [config-default#L27](https://github.com/uc-cdis/audit-service/blob/299d0c7ec49778af7d193ed5968641bf86a5bcf6/src/audit/config-default.yaml#L27-L27)

Feel free to correct me if I'm wrong!

Comment (paulineribeyre) [reply src/audit/pull_from_queue.py:87]: The default value is `None`, not `{}`

If someone set it to `{}`, it'd be fine, and if they set it to another type's null value such as `[]`, the config validation would error on line 103, which I think is acceptable behavior (erroring when the wrong type is used - although we could add a type check)

Comment (nss10) [reply src/audit/pull_from_queue.py:87]: If the value of `aws_sqs_config["aws_cred"]` is `None` then `config["AWS_CREDENTIALS"][None]` would throw an exception right? Is that what is expected?

Comment (paulineribeyre) [reply src/audit/pull_from_queue.py:87]: right we could add a check

**Pull request merged**